omnigibson/utils/grasping_planning_utils.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+import random
+from scipy.spatial.transform import Rotation as R, Slerp
+from math import ceil
+
+import omnigibson.utils.transform_utils as T
+from omnigibson.object_states.open_state import _get_relevant_joints
+from omnigibson.utils.constants import JointType, JointAxis
+from omni.isaac.core.utils.rotations import gf_quat_to_np_array
+
+REVOLUTE_JOINT_FRACTION_ACROSS_SURFACE_AXIS_BOUNDS = (0.4, 0.6)
+PRISMATIC_JOINT_FRACTION_ACROSS_SURFACE_AXIS_BOUNDS = (0.2, 0.8)
+GRASP_OFFSET = np.array([0, 0.05, -0.08])
+OPEN_GRASP_OFFSET = np.array([0, 0.05, -0.12])  # 5cm back and 12cm up.
+ROTATION_ARC_SEGMENT_LENGTHS = 0.05 # 10 cm
+
+def get_grasp_poses_for_object_sticky(target_obj):
+    bbox_center_in_world, bbox_quat_in_world, bbox_extent_in_base_frame, _ = target_obj.get_base_aligned_bbox(
+        visual=False
+    )
+
+    grasp_center_pos = bbox_center_in_world + np.array([0, 0, np.max(bbox_extent_in_base_frame) + 0.05])
+    towards_object_in_world_frame = bbox_center_in_world - grasp_center_pos
+    towards_object_in_world_frame /= np.linalg.norm(towards_object_in_world_frame)
+
+    grasp_quat = T.euler2quat([0, np.pi/2, 0])
+
+    grasp_pose = (grasp_center_pos, grasp_quat)
+    grasp_candidate = [(grasp_pose, towards_object_in_world_frame)]
+
+    return grasp_candidate
+
+# def get_grasp_poses_for_object_sticky(target_obj):
```

Comment:
is this outdated code? can we remove?

Reply:
It is currently unused but will be re-tried soon - I'm uncommenting it.